### DISCUSSION TITLE
Unit Test Cases for Karavan

### DISCUSSION THREAD

**Discussion by aijaiswa** [apache/camel-karavan#990] [Q&A] [answered]
Hi team, 

I am using karavan low code editor for creating integration, I want to know how I can create unit test case for my integrations. Also I want to publish code coverage. Kindly assist.

**Comment by mgubaidullin** [comment, marked as answer]
We have not yet implemented ny testing capabilities yet.
And we are not planning to make unit testing for Karavan but integration tests.
